vnext/template/cs-app/src/App.xaml.cs

```diff
@@ -49,6 +49,7 @@ protected override void OnLaunched(LaunchActivatedEventArgs e)
             frame.Navigate(typeof(MainPage), e.Arguments);
         }
 
+#if !USE_WINUI3
```

Comment:
See above feedback given about this pattern when looking at the C++ template.

Reply:
we had this logic in the app since we wanted to support having one app that could dual-target winui2 and winui3. I think we are no longer doing that, so this can be removed, and the right content should be stamped depending on whether the app is created with --useWinUI3 or not